### PR TITLE
Fixed a bug causing isImage() to not recognize upper-cased extensions

### PR DESCRIPTION
**Pull request by Seb-C**

### CODE REVIEW

**Comment by tonyyb** [comment]
:+1:

File: framework/classes/attachment.php [hunk 1]
```diff
@@ -60,7 +65,7 @@ public function __construct($attached, $config)
         }
 
         if (!empty($config['image']) && empty($config['extensions'])) {
-            $config['extensions'] = array('jpg', 'gif', 'png', 'jpeg');
+            $config['extensions'] = static::$image_extensions;
             unset($config['image']);
         }
```
Comment (Meroje) [inline framework/classes/attachment.php:67]: Yous should probably lowercase the config too `$config['extensions'] = array_map(array('\Str', 'lower'), static::$image_extensions)`

**Comment by smeeckaert** [comment]
You should put the configuration in the config/attachment file. It will be loaded by the system and can be checked by accessing $this->config instead of adding a static variable.

File: framework/classes/attachment.php (hunk 1 above)
Comment (Seb-C) [inline framework/classes/attachment.php:67]: @Meroje Good idea, but it's already done at line 75.

**Comment by Seb-C** [comment]
@smeeckaert I think that would be confusing: `$config['image']` can currently be set to `true` to auto-configure the allowed extensions, and `$config['extensions']` can be specified anyway, but is ignored if `$config['image']` is set.

The most logic thing to do would be to use `$config['extensions']` to override `static::$image_extensions` when `$config['image']` is also set, but may it break backward compatibility somehow ?!

**Comment by smeeckaert** [comment]
Oh okay, didn't quite understood the thing. Well maybe you could add a configuration key for this list like :

``` php
'image_extensions'=> array('jpeg',...)
```

That's a tiny bit of details, but there is already so many static variables and function in Nos, it's better not to add any more now, i think.

**Comment by smeeckaert** [comment]
:+1:

**Comment by shaoshiva** [comment]
:+1:

**Pull request merged**